papermill/iorw.py

```diff
@@ -255,9 +255,8 @@ def pretty_path(self, path):
 papermill_io.register("abs://", ABSHandler)
 papermill_io.register("http://", HttpHandler)
 papermill_io.register("https://", HttpHandler)
-gcs_handler = GCSHandler()
-papermill_io.register("gcs://", gcs_handler)
-papermill_io.register("gs://", gcs_handler)
+papermill_io.register("gcs://", GCSHandler())
```

Comment:
Do you want each end-point type (gs vs gcs) to share the cached client or is having two separate instances ok by default?

Reply:
I just checked with Cloud Storage team and @frankyn and we only support gs:// paths.

```
~ $ gsutil ls -al gs://dpe-sandbox/notebook.ipynb
     41946  2019-01-26T09:44:36Z  gs://dpe-sandbox/notebook.ipynb#1548495876399041  metageneration=1
TOTAL: 1 objects, 41946 bytes (40.96 KiB)
~ $ gsutil ls -al gcs://dpe-sandbox/notebook.ipynb
InvalidUrlError: Unrecognized scheme "gcs".
```
Documentation [here](https://cloud.google.com/storage/docs/listing-objects)

Would be interested in asking @bnaul the reason for adding gcs://, otherwise we can just keep gs://